index.html

```diff
@@ -7,19 +7,59 @@
 </head>
 <body>
   <video id="videojs-contrib-eme-player" class="video-js vjs-default-skin" controls>
-    <source src="//vjs.zencdn.net/v/oceans.mp4" type='video/mp4'>
-    <source src="//vjs.zencdn.net/v/oceans.webm" type='video/webm'>
+    <source src="/moose_encrypted.webm" type='video/webm'>
   </video>
   <ul>
     <li><a href="/test/">Run unit tests in browser.</a></li>
-    
+
   </ul>
   <script src="/node_modules/video.js/dist/video.js"></script>
   <script src="/dist/videojs-contrib-eme.js"></script>
   <script>
     (function(window, videojs) {
-      var player = window.player = videojs('videojs-contrib-eme-player');
-      player.contribEme();
+      // Example derived from http://www.html5rocks.com/en/tutorials/eme/basics/#clear-key
+
+      const KEY = new Uint8Array([
+        0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07,
+        0x08, 0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f
+      ]);
+
+      let player = window.player = videojs('videojs-contrib-eme-player');
+
+      // Convert Uint8Array into base64 using base64url alphabet, without padding.
+      let toBase64 = (u8arr) => {
+        return btoa(String.fromCharCode.apply(null, u8arr)).
+          replace(/\+/g, '-').replace(/\//g, '_').replace(/=*$/, '');
+      }
+
+      player.contribEme({
```

Comment:
Can we create two demos for this? This one and one showing the on-demand `player.src()` with keySystem options passed as part of the source object?